### PR TITLE
chore: add releaser

### PR DESCRIPTION
**Pull request by MacLikorne**

### CODE REVIEW

File: charts/Digital-Mobius/Chart.yaml [hunk 1]
```diff
@@ -0,0 +1,8 @@
+apiVersion: v2
+name: digital-mobius
+description: Automatically recycle not ready nodes on Digitial Ocean.
+type: application
+home: https://github.com/Qovery/digital-mobius
+version: 0.1.0
+appVersion: 0.1.0
+icon: https://github.com/Qovery/pleco/digital-mobius/main/assets/digital_mobius_logo.png
```
Comment (deimosfr) [inline charts/Digital-Mobius/Chart.yaml:8]: not pleco

File: .github/workflows/release.yml [hunk 2]
```diff
@@ -0,0 +1,90 @@
...
+          args: "run --timeout 5m0s"
+      -
+        name: Run GoReleaser
+        uses: goreleaser/goreleaser-action@v1
+        with:
+          version: latest
+          args: release --rm-dist
+        env:
+          GITHUB_TOKEN: ${{ secrets.GORELEASER_GITHUB_TOKEN }}
+      -
```
Comment (deimosfr) [inline .github/workflows/release.yml:64]: You should stop here if you don't want to have problems for not yet handled stuffs

Comment (MacLikorne) [reply .github/workflows/release.yml:64]: can I handle them now ?

File: charts/Digital-Mobius/Chart.yaml (hunk 1 above)
Comment (MacLikorne) [reply charts/Digital-Mobius/Chart.yaml:8]: corrected

**Pull request merged**